com.unity.netcode.gameobjects/Runtime/Metrics/NetworkMetrics.cs

```diff
@@ -75,6 +75,10 @@ private static string GetSceneEventTypeName(uint typeCode)
         {
             ShouldResetOnDispatch = true,
         };
+        private readonly Gauge m_RttToServerGauge = new Gauge(NetworkMetricTypes.Rtt.Id)
+        {
+            ShouldResetOnDispatch = true,
+        };
```

Comment:
I have a PR that renames `NetworkMetricTypes.Rtt` to `NetworkMetricTypes.RttToServer` to keep it consistent, please don't submit this version with the older name 🙏

Reply:
https://github.com/Unity-Technologies/com.unity.multiplayer.tools/pull/195

Reply:
Been merged.